packages/hardhat-chai-matchers/package.json

```diff
@@ -19,7 +22,8 @@
     "lint:fix": "yarn prettier --write && yarn eslint --fix",
     "eslint": "eslint 'src/**/*.ts' 'test/**/*.ts'",
     "prettier": "prettier \"**/*.{js,md,json}\"",
-    "test": "mocha --recursive \"test/**/*.ts\" --exit --reporter dot",
+    "test": "mocha --recursive \"test/**/*.ts\" --exit",
```

Comment:
Did you remove `--reporter dot` on purpose? I quite liked having it. Without it, every successful test run overflows my terminal's scrollback buffer.

Reply:
Yes, sorry, I did it to debug the OOM problem in the CI, but I don't mind adding it back.

Reply:
Np, I remove it for debugging too haha